compiler/qsc_data_structures/src/namespaces/tests.rs

```diff
@@ -25,37 +25,41 @@ fn test_tree_construction() {
         NamespaceTreeRoot
 
           children: [
-            ns1(id 11) {
+            Std(id 1) {
               children: [
-                nsc(id 14) {empty node},
-                nsb(id 13) {empty node},
-                nsa(id 12) {empty node},
+                Measurement(id 7) {empty node},
+                Canon(id 2) {empty node},
+                Intrinsic(id 6) {empty node},
```

Comment:
Why did these go from being programmatically named namespaces to explicitly named namespaces?

Reply:
It was previously the prelude + the programmatically inserted ones, and it still is. It's just the ordering that changed.